scene/resources/material.cpp

```diff
@@ -88,6 +88,37 @@ void Material::inspect_native_shader_code() {
 	}
 }
 
+RID Material::get_shader_rid() const {
+	RID ret;
+	if (GDVIRTUAL_CALL(_get_shader_rid, ret)) {
+		return ret;
+	}
+	ERR_FAIL_V_MSG(RID(), "Unimplemented function");
```

Comment:
Same for all other messages (adding "virtual" and dot at end of sentence):
```suggestion
	ERR_FAIL_V_MSG(RID(), "Unimplemented virtual function.");
```